config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml

```diff
@@ -85,33 +85,35 @@ presubmits:
       preset-aws-credential: "true"
       preset-bazel-scratch-dir: "true"
       preset-bazel-remote-cache-enabled: "true"
+    path_alias: "sigs.k8s.io/cluster-api-provider-aws"
     always_run: true
     optional: true
+    decorate: true
+    extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api
```

Comment:
Is the CAPI repo needed? Looks like the previous ci jobs weren't using it.

Reply:
sorry, deleted